njbrunner/covid-19
# Get user location

** Changes: ** Now pulling user's location when the home page loads if we don't currently have their location.

** Testing: **
- Create a new user
- When redirected to home page, you will be asked for location permissions
- Accept and see that latitude and longitude are displayed on the home page

- Change location permissions back to ask
- Create another new user
- When redirected to home page, you will be asked for location permissions
- Deny and see that alert is displayed informing the user why we need their location
